Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/logicalStructure.xhtml

```diff
@@ -20,10 +20,12 @@
         xmlns:p="http://primefaces.org/ui">
     <p:outputPanel id="logicalStructureHeader"
                    styleClass="metadata-column-menu">
-        <p:selectOneMenu styleClass="logicalStructureTitle" value="#{DataEditorForm.structurePanel.titleMetadata}"
+        <p:selectOneMenu id="logicalStructureTitle" value="#{DataEditorForm.structurePanel.nodeLabelOption}"
                          autoWidth="#{false}">
-            <f:selectItem itemLabel="#{msgs.typ}" itemValue="type"/>
-            <f:selectItems value="#{DataEditorForm.structurePanel.getTitleMetadataItems()}"/>
+            <f:selectItem itemLabel="#{msgs.typ}" itemValue="type" />
+            <f:selectItem itemLabel="#{msgs.title}" itemValue="title" />
+            <f:selectItem itemLabel="#{msgs.typ} - #{msgs.title}" itemValue="type+title" />
+            <!--<f:selectItem itemLabel="#{msgs.typ} - #{msgs.count}" itemValue="type+count" />-->
```

Comment:
Do you plan to implement this option as well? If not I would suggest to remove the comment. If we ever come around to adding the `count` option this should be (re-)added, in my opinion.

Reply:
Sorry. Forget to remove the comments that I left to remind me about the count option. I removed it.